pkg/apis/druid/v1alpha1/druid_types.go

```diff
@@ -200,6 +200,9 @@ type DruidNodeSpec struct {
 	// Optional
 	ReadinessProbe *v1.Probe `json:"readinessProbe,omitempty"`
 
+	// Optional
+	Lifecycle *v1.Lifecycle `json:"preStop,omitempty"`
```

Comment:
I  think you probably forgot to rename `preStop` to `lifeCycle` ?

Reply:
oh :(

Reply:
ill correct this